### ISSUE TITLE
Font does not render properly on macOS

### ISSUE THREAD

**Issue by mygizli04** [mircokroon/minecraft-world-downloader#201]
**Describe the bug**
It appears the world downloader cannot find the font that it's looking for, so it gets a different font which it's not compatible with.

**Steps to reproduce**
Open the jar file on macOS

**Log output**
```
2021-08-24 20:44:12.649 java[13797:369725] CoreText note: Client requested name ".SFNS-Regular", it will get Times-Roman rather than the intended font. All system UI font access should be through proper APIs such as CTFontCreateUIFontForLanguage() or +[NSFont systemFontOfSize:].
2021-08-24 20:44:12.649 java[13797:369725] CoreText note: Set a breakpoint on CTFontLogSystemFontNameRequest to debug.
2021-08-24 20:44:12.827 java[13797:369725] CoreText note: Client requested name ".SFNS-Regular", it will get Times-Roman rather than the intended font. All system UI font access should be through proper APIs such as CTFontCreateUIFontForLanguage() or +[NSFont systemFontOfSize:].
2021-08-24 20:44:13.356 java[13797:369757] CoreText note: Client requested name ".SFNS-Regular", it will get Times-Roman rather than the intended font. All system UI font access should be through proper APIs such as CTFontCreateUIFontForLanguage() or +[NSFont systemFontOfSize:].
2021-08-24 20:44:13.392 java[13797:369757] CoreText note: Client requested name ".SFNS-Regular", it will get Times-Roman rather than the intended font. All system UI font access should be through proper APIs such as CTFontCreateUIFontForLanguage() or +[NSFont systemFontOfSize:].
2021-08-24 20:44:13.400 java[13797:369757] CoreText note: Client requested name ".SFNS-Regular", it will get Times-Roman rather than the intended font. All system UI font access should be through proper APIs such as CTFontCreateUIFontForLanguage() or +[NSFont systemFontOfSize:]. 
...

```
**Screenshots**
![image](https://user-images.githubusercontent.com/42012824/130659874-d293d251-16e5-49c1-b4f9-d697e7270afd.png)

**Comment by k-willowhawk** [comment]
I tried installing the TTF for that font but still receive the java warning and garbled font.

I can at least get the app runnin on macOS, I can't get it running at all on Windows.

Can anyone running Windows app successfully post screenshots of the different tabs so I at least know what data to put into the fields?  I'm running on a server that whitelists players so I suspect it isn't allowing this MitM to connect.

**Comment by mygizli04** [comment]
I recommend you use the CLI options rather than using the GUI but here's all the tabs using the linux version in a VM (I actually ran into a bug where the Realms tab doesn't show the entire text no matter how big the window is):

<details>
<summary>Screenshots</summary>

![image](https://user-images.githubusercontent.com/42012824/131266111-775957d8-ef10-42e1-9764-9508e7d561c0.png)

![image](https://user-images.githubusercontent.com/42012824/131266162-dabee5fd-81e5-4649-87f0-194c6c5cbd0a.png)

![image](https://user-images.githubusercontent.com/42012824/131266172-c91dc875-dd97-445f-889c-cb927c8dd5cd.png)

![image](https://user-images.githubusercontent.com/42012824/131266177-d8e419d0-0f0f-450c-a658-8a7d4eab774e.png)

![image](https://user-images.githubusercontent.com/42012824/131266184-780a64a1-7cd6-4c4a-9a92-187dded7ab23.png)


</deatils>

**Comment by mircokroon** [comment]
Might be fixed on #210, can you verify @mygizli04?

**Comment by mygizli04** [comment]
Yeah, it looks like #210 fixed the issue.
![image](https://user-images.githubusercontent.com/42012824/132545648-f5cdcf8c-9ae7-48f4-9bfe-94f32a50d096.png)

**Comment by mircokroon** [comment]
Great, thanks!

**Comment by k-willowhawk** [comment]
> Might be fixed on #210, can you verify @mygizli04?

Confirmed working (both the font and authentication) on mine as well.
